src/libsystemd-network/sd-ipv4acd.c

```diff
@@ -25,18 +25,26 @@
 #include "string-util.h"
 #include "time-util.h"
 
-/* Constants from the RFC */
-#define PROBE_WAIT_USEC (1U * USEC_PER_SEC)
-#define PROBE_NUM 3U
-#define PROBE_MIN_USEC (1U * USEC_PER_SEC)
-#define PROBE_MAX_USEC (2U * USEC_PER_SEC)
-#define ANNOUNCE_WAIT_USEC (2U * USEC_PER_SEC)
-#define ANNOUNCE_NUM 2U
+/* Intervals from the RFC in seconds, need to be multiplied by the time unit */
+#define PROBE_WAIT 1U
+#define PROBE_MIN 1U
+#define PROBE_MAX 2U
+#define ANNOUNCE_WAIT 2U
+#define TOTAL_TIMEOUT 7U
```

Comment:
`TOTAL_TIMEOUT` sounds that it is a timespan.
Maybe, `TOTAL_STEPS` or `TOTAL_TIME_UNITS` or `TIME_UNITS_IN_TIMEOUT` or so is better?
Feel free to pick anything more suitable.